zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/nanoblob.dm

```diff
@@ -37,6 +37,8 @@
 
 	environment_smash = 1
 
+	agony_coefficient = 0.8 // rubbers aren't going to hurt a blob of goo very much
```

Comment:
Wouldn't this be the other way? If anything the way that LTL works (larger contact area at a slower speed) actually hurt a geletanous creature more?